packages/teams-js/src/public/runtime.ts

```diff
@@ -317,6 +312,13 @@ export const v1HostClientTypes = [
   HostClientType.teamsDisplays,
 ];
 
+export const v1HostClientTypes = [
+  ...v1NonMobileHostClientTypes,
+  HostClientType.android,
+  HostClientType.ios,
+  HostClientType.ipados,
+];
+
```

Comment:
I think this is more of a nit suggestion but maybe also making a constant for mobile host client types would look nice

```
const v1MobileHostClientTypes = [
  HostClientType.android,
  HostClientType.ios,
  HostClientType.ipados
];

export const v1HostClientTypes = [
  ...v1NonMobileHostClientTypes,
  ...v1MobileHostClientTypes,
];
```

Reply:
Sure, I can do that if it makes it clearer.